scripts/perf_mixed_workload.py

```
@@ -0,0 +1,392 @@
+                f"{failure['traceback']}"
+            )
+        raise primary_error.with_traceback(primary_traceback)
+    if len(cleanup_errors) == 1:
+        phase, error = cleanup_errors[0]
+        _attach_cleanup_failures(error, cleanup_errors)
+        error.add_note(f"Invocation cleanup phase: {phase}")
+        raise error.with_traceback(error.__traceback__)
+    if cleanup_errors:
+        for phase, error in cleanup_errors:
+            error.add_note(f"Invocation cleanup phase: {phase}")
+        grouped_error = BaseExceptionGroup(
```

Ruff (0.15.21)

[error] Undefined name `BaseExceptionGroup`. Consider specifying `requires-python = ">= 3.11"` or `tool.ruff.target-version = "py311"` in your `pyproject.toml` file.

(F821)